controllers/openstackcluster_controller.go

```diff
@@ -66,9 +67,14 @@ type OpenStackClusterReconciler struct {
 func (r *OpenStackClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, reterr error) {
 	log := ctrl.LoggerFrom(ctx)
 
+	mc := metrics.NewMetricPrometheusContext("snapshot", "create")
```

Comment:
I don't think we are doing a `snapshot create` here :-) also the r.Client.Get is no openstack request?!